### PR TITLE
torsocks: new package at 2.3.0

### PR DESCRIPTION
**Pull request by 6ZMQa2ND2KwrX2Z8JZcRzyQgDjP38g8V2BdX7NX**
## Description of package

Torsocks is a torifying wrapper that is primarily used to redirect all the network traffic of individual SOCKS-friendly applications through the Tor network. It also ensures DNS queries are handled correctly and explicitly blocks all UDP traffic from the application in question. Torsocks is the successor of tsocks and is still actively maintained.

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

/var/db/kiss/installed/torsocks/version
/var/db/kiss/installed/torsocks/sources
/var/db/kiss/installed/torsocks/manifest
/var/db/kiss/installed/torsocks/etcsums
/var/db/kiss/installed/torsocks/checksums
/var/db/kiss/installed/torsocks/build
/var/db/kiss/installed/torsocks/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man8/torsocks.8
/usr/share/man/man8/
/usr/share/man/man5/torsocks.conf.5
/usr/share/man/man5/
/usr/share/man/man1/torsocks.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/doc/torsocks/socks-extensions.txt
/usr/share/doc/torsocks/SOCKS5
/usr/share/doc/torsocks/DEBUG
/usr/share/doc/torsocks/ChangeLog
/usr/share/doc/torsocks/
/usr/share/doc/
/usr/share/
/usr/lib/torsocks/libtorsocks.so.0.0.0
/usr/lib/torsocks/libtorsocks.so.0
/usr/lib/torsocks/libtorsocks.so
/usr/lib/torsocks/libtorsocks.la
/usr/lib/torsocks/libtorsocks.a
/usr/lib/torsocks/
/usr/lib/
/usr/bin/torsocks
/usr/bin/
/usr/
/etc/tor/torsocks.conf
/etc/tor/
/etc/

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/torsocks/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+autoreconf -i
```
Comment (dylanaraps) [inline community/torsocks/build:3]: Does it not build without this?

Comment (6ZMQa2ND2KwrX2Z8JZcRzyQgDjP38g8V2BdX7NX) [reply community/torsocks/build:3]: No. :(

Comment (6ZMQa2ND2KwrX2Z8JZcRzyQgDjP38g8V2BdX7NX) [reply community/torsocks/build]: Fixed, thanks.

**Comment by dylanaraps** [comment]
Thanks :+1:

**Pull request merged**